django/db/backends/base/schema.py

```diff
@@ -831,33 +830,28 @@ def _alter_many_to_many(self, model, old_field, new_field, strict):
 
     def _create_index_name(self, model, column_names, suffix=""):
```

Comment:
As the method doesn't require anything from the model except its `_meta.db_table` could we alter its signature to accept a `table_name` instead? It would be more consistent with the `column_names` argument instead of an hypothetical `fields` one. This is a change planed in #6643.